### ISSUE TITLE
add support for ssh key auth

### ISSUE THREAD

**Issue by freeseacher** [mmumshad/ansible-playable#8]
Please add support for ssh key auth.

PS: Project looks perfect. tnx for sharing!

**Comment by mmumshad** [comment]
Thanks!

You mean SSH Key auth from playable to target Ansible host, correct?

**Comment by freeseacher** [comment]
yep. currently i can only provide password, filed for ssh key will be preferred.

**Comment by mmumshad** [comment]
Ok, I think the functionality is already there.. need to provide a user interface for it. I will look into it.

**Comment by fubarhouse** [comment]
+1 I can't even execute a playbook this on localhost via this one!

Can do it if I code up the play myself, but it's very unhappy ssh-ing into itself.

**Comment by dnlwgnr89** [comment]
Whilst not *preferable*, you can jump into the container with ; `docker exec -it 8bxxxxxxxxxx bash` and then create those files manually; `vi ~/.ssh/id_rsa` etc. Then it works fine with an SSH key :) Unsure if this would persist through a container restart however.

**Comment by dockerpoa** [comment]
Pls! Recompile your image with parameters, becose do not use config to especific user and pass variables in to config from inventories.

OLD: 

RUN apt-get update && apt-get install -y vim openssh-server


NEW:

RUN apt-get update && apt-get install -y vim openssh-server sshpass
